tests/unit_tests/storage/test_local_storage.py

```diff
@@ -436,11 +440,17 @@ def _inner(params):
             observations=st.dictionaries(
                 st.integers(min_value=0, max_value=200),
                 gen_observations,
-                max_size=1,
                 min_size=1,
+                max_size=1,
             ),
         ),
     ),
+    obs_time=st.lists(
+        st.datetimes(
+            min_value=datetime.strptime("1969-1-1", "%Y-%m-%d"),
```

Comment:
Perhaps better with  `datetime.fromisoformat("1969-01-01")`

Reply:
Fixed

Reply:
You need the leading zero in ISO-8601,
```
In [3]: datetime.strptime("1969-1-1", "%Y-%m-%d") == datetime.fromisoformat("1969-01-01")
Out[3]: True
```